tests/utils/job.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright The CloudNativePG Contributors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
+
+import (
+	"errors"
+	"fmt"
+
+	batchv1 "k8s.io/api/batch/v1"
+)
+
+// GetJob gets a Job by namespace and name
+func (env TestingEnvironment) GetJob(namespace, jobName string) (*batchv1.Job, error) {
+	wrapErr := func(err error) error {
```

Comment:
This function seems unused. Still, good idea to have it, and the linter is not complaining...

Reply:
I think this will be needed when we add E2e tests for replica cloning via Volume Snapshots.